scripts/list-parser-links.lua

```diff
@@ -0,0 +1,15 @@
+-- Execute as `nvim --headless -c "luafile ./scripts/list-parser-links.lua"
+local parsers = require 'nvim-treesitter.parsers'.get_parser_configs()
+local sorted_parsers = {}
+
+for k, v in pairs(parsers) do
+  table.insert(sorted_parsers, {name = k, parser = v})
+end
+
+table.sort(sorted_parsers, function(a,b) return a.name < b.name end)
```

Comment:
I don't know if this is a lua convention thing

```suggestion
table.sort(sorted_parsers, function(a, b) return a.name < b.name end)
```